WaterLog_Backend/Controllers/MonitorsController.cs

```diff
@@ -29,8 +29,16 @@ public MonitorsController(DatabaseContext context, IConfiguration config)
         // GET api/monitors
         [HttpGet]
         public async Task<ActionResult<IEnumerable<MonitorsEntry>>> Get()
-        {         
-            return await _db.Monitors.ToListAsync();
+        {
+            try
+            {
+                return await _db.Monitors.ToListAsync();
```

Comment:
I am not sure if this will catch it in the event where it returns an empty / null object